Assets/Scripts/Interactable.cs

```diff
@@ -58,6 +68,18 @@ void Start()
         foreach (SpriteRenderer renderer in this.myRenderers){
             renderer.material = defaultMaterial;
         }
+    } 
+
+    public void DisplayPrompt(bool select){
```

Comment:
Probably need to change the variable name select to something else here (maybe enabled?). It's giving some different syntax highlighting which I think means it's a special keyword or something.